tests/testsuite.at

```diff
@@ -223,7 +223,8 @@ m4_ifblank(m4_join([],ENV_CONFS), [
   m4_define([CONFS], [ENV_CONFS])
 ])
 
-m4_define([DEFAULT_CATEGORIES], [[Syntactic],
+m4_define([DEFAULT_CATEGORIES], [[Swig], 
```

Comment:
Remove Swig from this list so it's not a default test. A later PR will likely restore it when we've decided how to approach tests that rely on configure options.

Reply:
fixed